administrator/components/com_joomlaupdate/src/Model/NotificationModel.php

```diff
@@ -67,110 +82,77 @@ public function sendNotification($type, $oldVersion, $newVersion): void
             'url'        => Uri::root(),
         ];
 
-        // Send the emails to the Super Users
-        foreach ($superUsers as $superUser) {
-            $params = new Registry($superUser->params);
+        // Send emails to all receivers
+        foreach ($emailReceivers as $receiver) {
+            $params = new Registry($receiver->params);
             $jLanguage->load('com_joomlaupdate', JPATH_ADMINISTRATOR, 'en-GB', true, true);
             $jLanguage->load('com_joomlaupdate', JPATH_ADMINISTRATOR, $params->get('admin_language', null), true, true);
 
             $mailer = new MailTemplate('com_joomlaupdate.update.' . $type, $jLanguage->getTag());
-            $mailer->addRecipient($superUser->email);
+            $mailer->addRecipient($receiver->email);
             $mailer->addTemplateData($substitutions);
             $mailer->send();
         }
     }
 
     /**
-     * Returns the Super Users email information. If you provide a comma separated $email list
-     * we will check that these emails do belong to Super Users and that they have not blocked
-     * system emails.
+     * Returns the email information of receivers. Receiver can be any users who is not blocked.
```

Comment:
```suggestion
     * Returns the email information of receivers. Receiver can be any user who is not disabled.
```

fix grammar and changed to disabled